server/src/routers/userRouter.js

```diff
@@ -1,21 +1,29 @@
-const express = require('express')
-const path = require('path')
+const express = require("express");
+const path = require("path");
```

Comment:
Any particular reason for calling for replacing single quotes with double quotes?

Reply:
No, I do not remember changing it. I think I opted to use one of the templates I already had on my github and that is where it got changed. But I will make another commit where the kind of things will go back to normal and there will be no ';' either.